Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/QuadBendingFEMForceField.inl

```diff
@@ -700,15 +699,43 @@ void QuadBendingFEMForceField<DataTypes>::addForce(const core::MechanicalParams*
 template <class DataTypes>
 void QuadBendingFEMForceField<DataTypes>::addDForce(const core::MechanicalParams* mparams, DataVecDeriv& df, const DataVecDeriv& dx)  
 {
-  VecDeriv& df1 = *df.beginEdit();
-  const VecDeriv& dx1 = dx.getValue();
-  Real kFactor = sofa::core::mechanicalparams::kFactor(mparams);
+    auto df1 = sofa::helper::getWriteAccessor(df);
 
-  Real h=1;
-  df1.resize(dx1.size());
-  applyStiffnessSmall( df1, h, dx1, kFactor );
+    const VecDeriv& dx1 = dx.getValue();
+    Real kFactor = sofa::core::mechanicalparams::kFactor(mparams);
 
-  df.endEdit();
+    Real h = 1;
```

Comment:
Why is h always 1 ?

Reply:
I don't know, it was already here when this class was introduced in https://github.com/sofa-framework/sofa/pull/1745